src/docs/ArcCommanderManual/arc_isa_investigation.md

```diff
@@ -46,9 +48,10 @@ add sidebar: _sidebars/mainSidebar.md
 | Fax                       | The fax number of a person associated with the investigation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |                                                                                 |
 | Address                   | The address of a person associated with the investigation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      | Paul Ehrlich Straße 23, 67663 Kaiserslautern, Germany                           |
 | Affiliation               | The organization affiliation for a person associated with the investigation.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    | Department of M21 Measurements, Technische Universität Kaiserslautern, Germany  |
+| ORCID                     | The ORCID iD of the person   | https://orcid.org/XXXX-XXXX-XXXX-XXXX  |
```

Comment:
I don't think we should add ORCID here as such. (i) It's not an ISA term and (ii) it's not being asked during `arc i person register`

Reply:
> It can be added via --orcid XXXX-XXXX-XXXX-XXXX as a comment.

Cool. I forgot that the feature request made it back then! 
https://github.com/nfdi4plants/arcCommander/issues/55